src/integrators/sdirk_integrators.jl

```diff
@@ -224,15 +223,15 @@ end
     W = I - dto2*J
   else
     J = ForwardDiff.derivative(uf,uprev)
-    W = 1 - dto2*J
+    W = @. 1 - dto2*J
```

Comment:
So I'll change it there as well such that W is already calculated inside the if-else expression - once with identity matrix `I` and once just as a number?